style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+.nav-bar {
+  height: 72px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 1.5rem;
+  background-color: white;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  align-items: center;
+  color: #6070ff;
+}
+
+.logo a {
+  color: #6070ff;
+}
+
+.logo-dot {
+  font-size: 50px;
+  color: rgb(231, 38, 0);
+}
+
+.hero-container {
+  background-image:
+    url("/images/Hero-Mobile-Background/Ellipse\ 3.svg"),
+    url("/images/Hero-Mobile-Background/Ellipse\ 4.svg"),
+    url("/images/Hero-Mobile-Background/Ellipse\ 5.svg"),
+    url("/images/Hero-Mobile-Background/Ellipse\ 8.svg"),
+    url("/images/Hero-Mobile-Background/Ellipse\ 11.svg"),
+    url("/images/Hero-Mobile-Background/Ellipse\ 12.svg"),
+    url("/images/Hero-Mobile-Background/Ellipse\ 13.svg"),
+    url("/images/Hero-Mobile-Background/Ellipse\ 14.svg"),
+    url("/images/Hero-Mobile-Background/Ellipse\ 15.svg"),
+    url("/images/Hero-Mobile-Background/Rectangle\ 5.svg"),
+    url("/images/Hero-Mobile-Background/Rectangle\ 7.svg");
```

Comment:
- [ ] [OPTIONAL] Great job implementing the background images for this section. I would recommend that you add a dot "." in front of the `/images/` file path as different operating systems resolve paths differently. Below I have included for you what I actually mean.

```
background-image:
    url("./images/Hero-Mobile-Background/Ellipse\ 3.svg"),
    url("./images/Hero-Mobile-Background/Ellipse\ 4.svg"),
    url("./images/Hero-Mobile-Background/Ellipse\ 5.svg"),
    url("./images/Hero-Mobile-Background/Ellipse\ 8.svg"),
    url("./images/Hero-Mobile-Background/Ellipse\ 11.svg"),
    url("./images/Hero-Mobile-Background/Ellipse\ 12.svg"),
    url("./images/Hero-Mobile-Background/Ellipse\ 13.svg"),
    url("./images/Hero-Mobile-Background/Ellipse\ 14.svg"),
    url("./images/Hero-Mobile-Background/Ellipse\ 15.svg"),
    url("./images/Hero-Mobile-Background/Rectangle\ 5.svg"),
    url("./images/Hero-Mobile-Background/Rectangle\ 7.svg");
```

**This will help you not run into trouble with reviewers in future for background images not showing on their operating systems**

Reply:
Thanks so much. I would never have known.